### ISSUE TITLE
Unknown symbols appear in Pillow import

### ISSUE THREAD

**Issue by tsulala** [microsoft/pyright#2004]
**Describe the bug**
It should be a correct import, but when I import the Pillow module, it appears with an unknown symbol.

**To Reproduce**
```python
from PIL import Image  # ok
from PIL import ImageCms, ImageTk, ImageTransform  # no, there are others...
```

**Expected behavior**
I was hoping that Pillow's type hints would be supported.

**Screenshots or Code**
![image](https://user-images.githubusercontent.com/39614590/122505341-2061ef80-d037-11eb-8ae1-fae28772a142.png)

**VS Code extension or command-line**
Pylance version: `2021.6.2`

**Additional context**
I think the reason for this issue is that the Pillow stubs in typeshed are not included in Pyright and refer to the old Pillow stubs in  [microsoft/python-type-stubs](https://github.com/microsoft/python-type-stubs).

The latest stub for Pillow has been recently added to `typeshed`. (python/typeshed [#5594](https://github.com/python/typeshed/pull/5594))

**Comment by erictraut** [comment]
I'm going to transfer this issue to the pylance-releases repo because it's not related to core type checking functionality but instead relates to type stubs that ship with pylance.
